Features.md

```diff
@@ -0,0 +1,17 @@
+# Feature
```

Comment:
This file is being added in two different branches, this will cause a conflict when merged. Either add this file in this branch or the other branch. But, do not do this in both branches.